server/mailet/integration-testing/src/test/java/org/apache/james/smtp/SmtpRandomStoringTest.java

```diff
@@ -210,14 +202,17 @@ private void checkMailboxesHaveBeenFilled(Collection<IMAPMessageReader> connecti
     }
 
     private LongStream numberOfAUserMessages(IMAPMessageReader imapMessageReader) {
-        return MAILBOXES
-            .stream()
+        return userMailboxesNames()
             .mapToLong(mailbox -> numberOfMessagesInMailbox(imapMessageReader, mailbox));
     }
 
+    private Stream<String> userMailboxesNames() {
+        return IntStream.range(0, PER_USER_MAILBOX_COUNT)
+            .mapToObj(i -> RandomStoring.PROVISIONNED_MAILBOX_PREFIX + i);
```

Comment:
you are going to generate mailboxes with too specific names. As we use this mailet for benchmarking it's not a good idea to use a pattern like this for names. However you could generate real random names and always use the same seed to ensure you don't need to store the result of random generation.